### ISSUE TITLE
Client registration policies not available in Keyclok 19.0.1

### ISSUE THREAD

**Issue by souravs17031999** [keycloak/keycloak#14808]
### Describe the bug

Prior to keycloak 19, there was a separate tab and process for creating client registration policies, we need those policies in Keycloak 19 as well for OIDC dynamic client registrations.
I am unable to find any tab for creating those.
Even as per Keycloak latest docs, it says there's a plan to remove these "client registration policies" and migrate to "client policies" but I can't see anywhere where it says it has been deprecated/removed already and also following policies which were available earlier for ex. "Max Clients Policy" are not available now in client policies.
![qa](https://user-images.githubusercontent.com/33771969/194738136-19408453-d02a-45b6-8090-8b7540c8e51b.PNG)

### Version

19.0.1

### Expected behavior

There should be a client registration policies tabs for managing client policies related to dynamic client registrations.
If those are not available, then existing client registration policies should be migrated and available in client policies tabs.

### Actual behavior

Client registration policies is not available in admin console.

**Comment by stianst** [comment]
@mposolda @ssilvert seems the old client registration policies may be missing in the new admin console? AFAIK those have not been fully deprecated yet?

**Comment by souravs17031999** [comment]
@stianst It seems only way to update/view existing policies is by admin API's ?

**Comment by souravs17031999** [comment]
Ok, thanks @ssilvert , I can try with that one.

But still, I see documentation for these parts should be updated since the one that is present on the latest one still shows/says the things that could have been true for older versions of KC (say 18 or earlier).

Also, I am observing one tricky behaviour in KC19, I am able to delete client created using dynamic registration flow using only a single realm-management role "create-client" , is this intented behaviour ? @stianst .
The documentation says something else.

**Comment by souravs17031999** [comment]
Update: It worked with the solution suggested , thanks @ssilvert :)

Still, I will wait for my above questions to be answered (if possible).

**Comment by mposolda** [comment]
@stianst Yes, they are not fully deprecated and AFAIK there are some things, which are supported by client registration policies, which don't yet work with "client policies".  AFAIK for example dynamic client registration with anonymous mode (without initial access token), which is needed for OIDC certification etc. 

@ssilvert @stianst If it is not huge amount of work, I suggest to re-create those in the new admin console as well.

@souravs17031999 If you are able to remove client with bearer client of user (or service account) without `manage-client` role, then it looks like a bug. I suggest to create separate issue for that one. The "domain/host name" with Trusted Hosts policy will need to contain * at the beginning AFAIR. Something like "*.mydomain.com" . Did you tried this option?

**Comment by ssilvert** [comment]
@mposolda @stianst In the new admin console, we put `Initial access token` under `Clients` instead of `Realm settings`.

Which implementation makes more sense to you?

`Clients --> Initial access token`
`Clients --> Client registration policies`

or

`Clients --> Client Registration --> Initial access token`
`Clients --> Client Registration --> Client registration policies`

Basically, I'm asking if we should have a `Client Registration` tab like in the old console or put them on main tabs off of `Clients`.  Right now, `Clients` only has two tabs, so there is plenty of room.

**Comment by ssilvert** [comment]
I guess a third option is to duplicate what we had in the old console, which is to put both under `Realm settings`.  But `Realm settings` is already quite crowded.

**Comment by mposolda** [comment]
@ssilvert Whatever options makes more sense to you (or someone else with good UX feeling according your preference) :-) And/or whatever option is easiest to implement and will be easiest to change in the future (once we decide to remove `Client registration policies`).

I am 50/50 between option (1) and (2). I guess (3) is not good as it means more work for you with moving `Initial Access Token` to realm settings?

**Comment by ssilvert** [comment]
My vote is for number 1.  But it's really not hard at all to move these around.  Whatever makes the most sense to the user is what I'd like to see.

**Comment by souravs17031999** [comment]
@mposolda Thanks for replies. 
So, For the first part, yes, I was able to delete client without ```manage-client``` role but I will check once more and create a separate issue for this one.

Now, for the second part regarding trusted hosts policy, I tried with "*.mydomain.com" option, It didn't worked.
From the logs, I can see it says something like ```Failed to verify remote host : xx.xxx.xx.xx```
